File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) throws Exception {
      
    Scanner sc = new Scanner(System.in);
    int N = sc.nextInt();
    sc.nextLine();

    String nums = sc.nextLine();
    String arr[] = nums.split(" ");
    double sum = 0;
    double max = 0;
    for (int i = 0; i < N; i++) {
      float num = Float.parseFloat(arr[i]);
      sum += num;
      if (max < num) {
        max = num;
      }
    }
    System.out.println(sum / N * 100 / max);
  }
}

File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>;
using namespace std;

int main() 
{    
    int N;
    cin >> N;
    int A[1000];
    
    for (int i = 0; i < N; i++)
    {
        cin >> A[i];
    }
    
    long sum = 0;
    long max = 0;

    for (int i = 0; i < N; i++)
    {   
        sum += A[i];
        if (max < A[i]) {
            max = A[i];
        }
    }
    double result = sum * 100.0 / max / N;

    cout << result;
    return 0;
}